apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -337,6 +337,8 @@ export class Utilities {
     }
 
     const environment: IEnvironment = Utilities._createEnvironmentForRushCommand(options.initCwd);
+    // load the local node_modules/.bin directory into the PATH
+    environment.PATH = `${path.resolve(options.workingDirectory, 'node_modules', '.bin')}:${environment.PATH}`;
```

Comment:
I see that **EventHooksManager.ts** sets `options.workingDirectory` to `process.cwd()` when it calls this function.

Reply:
Whereas **GlobalScriptAction.ts** uses `workingDirectory: this.rushConfiguration.rushJsonFolder` which seems more intuitive.  However in that case this `PATH` isn't going to be correct.

Reply:
Ah good point. Glad you caught that.